_posts/2019-01-23-developing-rails-applications-on-docker.md

```diff
@@ -0,0 +1,207 @@
+---
+layout: blog
+title: Developing Rails Applications on Docker
+date: 2019-01-23 09:48 +0300
+categories: developer
+published: false
+author: Melvin Atieno, Ngari Ndung'u, Tom Nyongesa
+blog-image: rails-docker/railsondocker.png
+intro: |
+  Rails and Docker are important components in the development processes at Zegetech.
+  Rails is our chosen platform for most of what we build, and docker provides pain-free environment management both for development and in production.
+  We have previously covered these two technologies separately, and this post covers the sweet spot at their intersection.
+  We will take you through the process of configuring a rails development environment on docker, and configure a postgres database for it.
+---
+
+![Rails on docker](/assets/images/blog/{{page.blog-image}}){:.img-responsive .center}
+
+{{page.intro | markdownify}}
+
+I would suggest you have a look at the following posts, if this is the first time you are encountering Docker and/or Rails:
+- [What and Why Ruby on Rails](2018-10-17-why-ruby-on-rails.md)
+- [What and Why Docker](2018-10-29-what-and-why-docker.md)
+- [Developing with Docker](2018-11-08-developing-with-docker.md)
+
+Lets start by defining what our end goal is. We want:
+- A docker image with an up-to-date and ready to use rails development environment,
+- Access to our database of choice(postgres), and
+- A means to easily run the setup
```

Comment:
You need to add prerequisites needed for this process to work. Eg. you need docker -v to run etc